uint32 control_mode
uint32 input_mode
float32 input_pos
float32 input_vel
float32 input_torque
